integration_tests/src/test/scala/com/nvidia/spark/rapids/JoinsSuite.scala

```diff
@@ -97,4 +99,45 @@ class JoinsSuite extends SparkQueryCompareTestSuite {
     mixedDfWithNulls, mixedDfWithNulls, sortBeforeRepart = true) {
     (A, B) => A.join(B, A("longs") === B("longs"), "LeftAnti")
   }
+
+  test("broadcast hint isn't propagated after a join") {
```

Comment:
How are these integration tests?  They don't fit with that model, and look much more like "unit" tests.  Could you please move them to the tests directory instead of integration_tests

Reply:
Thanks @revans2  for reviewing. My bad, since all the join tests were in this file, I added these here as well not realizing these do not qualify here. I have moved these tests to  a new file under tests directory. Please take a look and let me know if it's okay to keep there  or should be put in any existing file.